### PR TITLE
add test scripts for issue 3857 [internal review]

### PR DESCRIPTION
**Pull request by VjKlepikov**
ATF Test Scripts to check #[FORDTCN-12637](https://luxproject.luxoft.com/jira/browse/FORDTCN-12637)

This PR is **[ready]** for review.

### Summary
Add test scripts for issue 3857

### ATF version
latest

### Changelog
The new script is added
The test set is updated with new scripts

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Defects/8_2/3857_Crash_on_Ignition_Off_during_registering_Apps.lua [hunk 1]
```diff
@@ -0,0 +1,109 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3857
+---------------------------------------------------------------------------------------------------
+-- Description: The crash occurs on Ignition OFF during registering Apps
+---------------------------------------------------------------------------------------------------
+-- Preconditions:
+-- 1. SDL and HMI are started
+-- In case:
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/8_2/3857_Crash_on_Ignition_Off_during_registering_Apps.lua]:
```suggestion
-- Steps:
```

File: test_scripts/Defects/8_2/3857_Crash_on_Ignition_Off_during_registering_Apps.lua [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+-- 1. Register 5 Apps during Ignition OFF
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/8_2/3857_Crash_on_Ignition_Off_during_registering_Apps.lua]:
```suggestion
-- 1. Mobile app starts registration of 5 apps
-- 2. Ignition off is performed during apps registration
```

File: test_scripts/Defects/8_2/3857_Crash_on_Ignition_Off_during_registering_Apps.lua [hunk 3]
```diff
@@ -0,0 +1,109 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3857
+---------------------------------------------------------------------------------------------------
+-- Description: The crash occurs on Ignition OFF during registering Apps
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/8_2/3857_Crash_on_Ignition_Off_during_registering_Apps.lua]: Please define description from SDL perspective lile `There is no SDL crash on ignition off ....`

File: test_scripts/Defects/8_2/3857_Crash_on_Ignition_Off_during_registering_Apps.lua [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+          for i = 1, common.mobile.getAppsCount() do
+            common.mobile.deleteSession(i)
+          end
+        end)
+    end)
+  common.run.wait(3000)
+end
+
+local function registerAppIgn()
+  local isFailed = false
```
Comment (GetmanetsIrina) [inline test_scripts/Defects/8_2/3857_Crash_on_Ignition_Off_during_registering_Apps.lua]: Propose to rename to isDisconnected

**Review by GetmanetsIrina — CHANGES_REQUESTED** [review]

**Comment by VjKlepikov** [comment]
@GetmanetsIrina Please find the update in [8cd6c](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/196/commits/8cd6c811ca1e81119a009fedeca50b6612a7a31b)

**Review by GetmanetsIrina — APPROVED** [review]

**Comment by VjKlepikov** [comment]
move to open

**Pull request closed without merging**